src/styles/atoms/_btn.scss

```diff
@@ -169,6 +169,24 @@
     width: 100%;
 }
 
+//  = FLAT BUTTON
+//  ---------------------------------------------------------------------------
+@mixin dc-btn--flat {
+    @include dc-link;
+
+    background: transparent;
+    border: 0;
+}
+
+@mixin dc-btn--flat--no-touch {
+    &:hover {
+        border: 0;
```

Comment:
Yeah that's what I meant - I just tested it locally and it keeps the buttons at the same height